dask_glm/tests/test_admm.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from dask import persist
+import dask.array as da
+import numpy as np
+
+from dask_glm.logistic import admm, local_update
+from dask_glm.utils import make_y
+
+
+@pytest.mark.parametrize('N', [1000, 10000])
```

Comment:
For just `local_update` test, this maybe an overkill. Thoughts? ` test_admm_with_large_lamduh` should capture the high level tests correct?